analysis_service/settings.py

```diff
@@ -37,6 +37,7 @@
 
     # Third party apps
     'django_jinja',
+    'whitenoise.runserver_nostatic',
```

Comment:
I think the wsgi file should also drop the use of the separate WSGI app, since when using the whitenoise middleware that's not needed anymore.